src/svelte/ChecklistGroup.svelte

```diff
@@ -31,21 +43,23 @@
     </div>
     <div class="space" />
     <div class="count">{group.todos.length}</div>
-    <div class="collapse" on:click={() => onToggle(group.groupId, "page")}>
+    <button class="collapse" on:click={() => onToggle(group.groupId, "page")} title="Toggle Group">
       <Icon name="chevron" direction={isCollapsed ? "left" : "down"} />
-    </div>
-  </div>
-  {#if !isCollapsed}
-    {#each group.todos as item}
-      <ChecklistItem {item} {lookAndFeel} {app} />
-    {/each}
-  {/if}
-</div>
+    </button>
+  </header>
+  <ul>
+    {#if !isCollapsed}
+      {#each group.todos as item}
+        <ChecklistItem {item} {lookAndFeel} {app} />
+      {/each}
+    {/if}
+  </ul>
+</section>
```

Comment:
These changes make it so that you can style things by the tag name (i.e., `section` instead of `.group`) but didn't want to step all over code you'd already written. Not trying to take over all your hard work!

Reply:
No this is awesome. I am actually working on a branch to use `markdown-it` to parse the checklist items so there is full markdown support rather than trying to parse things myself. This will render into raw HTML using tags so this is a step in the same direction I'm taking currently

Reply:
Oh nice! That'll be awesome to have!